src/redis-cli.c

```diff
@@ -7768,24 +7777,29 @@ unsigned long long sendSync(redisContext *c, char *out_eof, int send_sync) {
     return strtoull(buf+1,NULL,10);
 }
 
-static void slaveMode(int is_psync) {
+static void slaveMode(int send_sync) {
     static char eofmark[RDB_EOF_MARK_SIZE];
     static char lastbytes[RDB_EOF_MARK_SIZE];
     static int usemark = 0;
-    unsigned long long payload = sendSync(context,eofmark,!is_psync);
+    static int out_full_mode;
+    unsigned long long payload = sendSync(context,eofmark,&out_full_mode,send_sync);
     char buf[1024];
     int original_output = config.output;
-    char *command = is_psync ? "PSYNC" : "SYNC";
+    char *command = out_full_mode ? "SYNC" : "PSYNC";
```

Comment:
that's not necessarily right. it could be a +FULLRESYNC that's a result of a PSYNC.
i don't mind dropping that info (command name) or replace it with something else that may be useful, but let's not print misleading text.

Reply:
yes, i change it to `Full resync` and `Partial resync`, look can't go wrong

Reply:
yes, i change it to `Full resync` and `Partial resync`, look can't go wrong